package.json

```diff
@@ -4,7 +4,10 @@
   "description": "- Nodejs 7.x\r - npm: discord.js",
   "main": "index.js",
   "scripts": {
-    "test": "mocha"
+    "start": "node src/bot.js",
+    "test": "mocha",
+    "start:docker": "docker-compose up --build -d",
```

Comment:
Just one question, we only use one container now. How will this work when we have multiple containers(like digbot + database)? Will it console log everything from all containers?

Reply:
I renamed some commands and added a few:
- docker:start will build and start the containers in detached mode
- docker:logs will give a realtime view of the digbot containers logs
- docker:stop will stop the containers and remove them from docker

The reason behind this is that when you run it without detach and use ctl+c to stop the container it will exit with an error and you still have to use the down command. An extra command could be added to combine start and logs scripts.

Reply:
Well before I believe it was just run from a separate directory, so detached or not it wouldn't have mattered, I don't remember though.

My main complaint on the function of the scripts is the lack of logs so if we could just have an ```up``` script to bring it up with logs and ```down``` to bring it down that could work